Readme.md

```diff
@@ -0,0 +1,153 @@
+
+** TRACY HTML/CSS & JavaScript Today's to do list** is a basic website that allows users to note down plan for the day in a list .
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- LIVE DEMO -->
+ 
+## 🚀 Live Demo <a name="live-demo"></a>
+ 
+- [<a href="#">LIVE DEMO]
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- GETTING STARTED -->
+ 
+## 💻 Getting Started <a name="getting-started"></a>
+ 
+To get a local copy up and running, follow these steps.
+ 
+### Prerequisites
+ 
+In order to run this project you need:
+ 
+- A computer.
+- Internet access.
+- A modern web browser.
+ 
+### Setup
+ 
+Clone this repository to your desired folder:
+ 
+```sh
+  cd my-folder
+  git@github.com:Tracy-miranja/TO-DO-LIST.git
+```
+ 
+### Install
+ 
+Install this project with:
+ 
+```sh
+  cd my-folder
+  npm-install
+```
+### Usage
+ 
+To run the project, open index.html in your text editor,right-click and open with live server,it will be opened in your web browser.
+ 
+<!-- AUTHORS -->
+ 
+## 👥 Authors <a name="author"></a>
+ 
+ 
+👤 **Tracy & Queenter**
+ 
+- GitHub: [@githubhandle](https://github.com/Tracy-miranja)
+- Twitter: [@twitterhandle](https://twitter.com/tracymiranja)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/tracymiranja)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- FUTURE FEATURES -->
+ 
+## 🔭 Future Features <a name="future-features"></a>
+ 
+- [ ] **Web page header**
+- [ ] **Web page body**
+- [ ] **Web page footer**
+- [ ] **Desktop design**
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Key FEATURES -->
+
+- [ ] **Homepage**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- SUPPORT -->
+ 
+## ⭐️ Show your support <a name="support"></a>
+ 
+If the project was great as your view, don't hesitate to share and give it a star.
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- ACKNOWLEDGEMENTS -->
+ 
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+ 
+I would like to thank  all my Mricronauts partners for support and microverse for giving me this design template to work on.
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+<!-- LICENSE -->
+ 
+## 📝 License <a name="license"></a>
+ 
+This project is [MIT](https://github.com/Tracy-miranja/TO-DO-LIST/blob/add-license-1/LICENCE licensed.
```

Comment:
- [ ] [OPTIONAL] Please add closing bracket ")" after the link for MIT to become a link itself. This would help you making your readme more professional.